src/frontend/package.json

```diff
@@ -34,6 +34,7 @@
     "@typescript-eslint/eslint-plugin": "8.32.0",
     "@typescript-eslint/parser": "8.32.0",
     "eslint": "8.57.0",
+    "prosemirror-model": "1.25.0",
```

Comment:
It will be updated next week. Is it what you want ?

Reply:
Do you mean we should add it in `renovate` ? Yes you're right I think better to do that.

Reply:
Yes I mean, should we add it to renovate indeed